low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -214,10 +213,65 @@ def get_platform_sensors_info(self):
                 )
         return response
```

Comment:
Line 182 
>        status = 'Ok' if reading[2] == 'ok' else 'NA'

Can you change status to 'OK' to be consistent with out resource status, this is to correct one mistake we made in PR #420

Reply:
done